src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -95,6 +98,9 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
       <div className="AppCard__header">
         <div className="AppCard__header__logo">
           <img src={logo || '/fallback.svg'} alt={`${name} logo`} />
+          {!isVerified ? (
+            <img src={SpannerSVG} className="AppCard__header__logo__dev-icon" alt="Dev mode icon" />
+          ) : null}
```

Comment:
Nit pick: you don't need a ternary in this case.
```suggestion
          {!isVerified && (
            <img src={SpannerSVG} className="AppCard__header__logo__dev-icon" alt="Dev mode icon" />
          )}
```